person.rb

```diff
@@ -23,7 +27,9 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Great job creating a method for adding rentals 👏🏻 👏🏻 However, the `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association. it should look like this 👍🏻 

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```